airbyte-webapp/src/components/ui/Toast/Toast.tsx

```diff
@@ -1,86 +1,64 @@
-import { faTimes } from "@fortawesome/free-solid-svg-icons";
+import { faCheck, faExclamation, faTimes } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import classNames from "classnames";
 import React from "react";
-import styled, { keyframes } from "styled-components";
-
-import { H5 } from "components/base/Titles";
 
 import { Button } from "../Button";
-import { ErrorSign } from "./ErrorSign";
+// eslint-disable-next-line css-modules/no-unused-class
```

Comment:
Why does it think there's unused classes?

Reply:
I think because I am not using error, warning classes directly rather they are applied based on the toast type.

Reply:
Not worth making a mapping to tell it otherwise. 👍